.travis.yml

```diff
@@ -3,21 +3,22 @@ sudo: false
 matrix:
   include:
     - scala: 2.10.7
-      script: ./sbt ++$TRAVIS_SCALA_VERSION clean test mimaReportBinaryIssues
+      jdk: openjdk8
+      script: sbt ++$TRAVIS_SCALA_VERSION clean test mimaReportBinaryIssues
 
     - scala: 2.11.12
-      jdk: oraclejdk8
-      script: ./sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport mimaReportBinaryIssues
+      jdk: openjdk8
+      script: sbt ++$TRAVIS_SCALA_VERSION clean coverage test coverageReport mimaReportBinaryIssues
       after_success:
         - bash <(curl -s https://codecov.io/bash)
 
-    - scala: 2.12.6
-      jdk: oraclejdk8
-      script: ./sbt "+++$TRAVIS_SCALA_VERSION clean" "+++$TRAVIS_SCALA_VERSION test" # "+++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues"
+    - scala: 2.12.8
```

Comment:
saw that you had issues with `2.12.10` in 2dfc5bd; curious if the same thing happens with `2.12.9`

Reply:
Yes happens to both 2.12.9 & 2.12.10, have to stick with 2.12.8.